### ISSUE TITLE
Return type of vue-hooks

### ISSUE THREAD

**Issue by c-orter** [victorgarciaesgi/simple-graphql-to-typescript#3]
First things first: this project looks amazing and I am super excited to use it!

I have a question though and was hoping you could provide some clarification. It looks like the wrong type is being passed into the useQuery call. If right now I have a User model in my graphql schema the code generated will be
```typescript
useQuery<UseQueryOptions<{ users: User[] }, usersArgs>>(query, variables, options);
```

But according to the code generated by the react-hooks generator and from what I can tell from the vue-apollo documentation it should be
```typescript
useQuery<{ users: User[] }, usersArgs>(query, variables, options);
```

This means that if I try to use the useUsers hook the returned result has the wrong type but the correct contents.

Thanks,
Carter

**Comment by victorgarciaesgi** [comment]
Hi @c8r821, thanks for the feedback! I will check this right now and keep you updated :), it looks like some mistake from my side, I don't have much free time to write good tests :(

**Comment by victorgarciaesgi** [comment]
@c8r821 Can you confirm this is the correct output? (I also removed `variables` argument when the query has no arguments)

```ts
export const useUsers = (
  fragment: string | DocumentNode,
  options?:
    | UseQueryOptions<{ users: IUserModel[] }>
    | Ref<UseQueryOptions<{ users: IUserModel[] }>>
    | ReactiveFunction<UseQueryOptions<{ users: IUserModel[] }>>
) => {
  const { isString, isFragment, fragmentName } = guessFragmentType(fragment);
  const query = sgtsQL`
      query users  {
        users {
          ${isString ? fragment : '...' + fragmentName}
        }
      } ${isFragment ? fragment : ''}
      `;

  return useQuery<{ users: IUserModel[] }>(query, null, options);
};
```

**Comment by c-orter** [comment]
Looks good to me! Thanks for the quick feedback and the awesome tool!

**Comment by victorgarciaesgi** [comment]
@c8r821 Thanks! It's published on `0.6.3` :)
